de_web_plugin.cpp

```diff
@@ -3061,6 +3106,7 @@ void DeRestPluginPrivate::addSensorNode(const deCONZ::Node *node, const SensorFi
     }
     else if ((node->nodeDescriptor().manufacturerCode() == VENDOR_OSRAM_STACK) || (node->nodeDescriptor().manufacturerCode() == VENDOR_OSRAM))
     {
+        // TODO: Heiman sensors also use VENDOR_OSRAM_STACK
```

Comment:
Indeed we should use data from basic cluster here.
I think VENDOR_OSRAM_STACK should really be VENDOR_MARVEL (if I remember correctly, need to check).